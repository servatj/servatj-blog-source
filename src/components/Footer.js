import React from 'react'

import {rhythm} from '../utils/typography'

const Footer = () => {
  return (
    <footer
      style={{
        marginTop: rhythm(2.5),
        paddingTop: rhythm(1),
        color: '#fff',
      }}
    >
      <a
        href="https://mobile.twitter.com/servatj"
        target="_blank"
        rel="noopener noreferrer"
      >
        twitter
      </a>{' '}
      &bull;{' '}
      <a
        href="https://github.com/servatj"
        target="_blank"
        rel="noopener noreferrer"
      >
        github
      </a>{' '}
      &bull;{' '}
      <a
        href="https://www.linkedin.com/in/josep-servat-lorca-384ba757/"
        target="_blank"
        rel="noopener noreferrer"
      >
        linkedin
      </a>{' '}
      &bull; <a href="https://discord.gg/vbE4MaJ">Discord</a>
    </footer>
  )
}

export default Footer
