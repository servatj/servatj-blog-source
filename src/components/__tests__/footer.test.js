/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'
import {StaticQuery} from 'gatsby'
import Footer from '../Footer'

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({render}) =>
    render({
      site: {
        siteMetadata: {
          title: `Default Starter`,
        },
      },
    }),
  )
})

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
