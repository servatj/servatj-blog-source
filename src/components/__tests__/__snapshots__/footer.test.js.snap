// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  style={
    Object {
      "color": "#fff",
      "marginTop": "4.375rem",
      "paddingTop": "1.75rem",
    }
  }
>
  <a
    href="https://mobile.twitter.com/servatj"
    rel="noopener noreferrer"
    target="_blank"
  >
    twitter
  </a>
   
  •
   
  <a
    href="https://github.com/servatj"
    rel="noopener noreferrer"
    target="_blank"
  >
    github
  </a>
   
  •
   
  <a
    href="https://www.linkedin.com/in/josep-servat-lorca-384ba757/"
    rel="noopener noreferrer"
    target="_blank"
  >
    linkedin
  </a>
   
  •
   
  <a
    href="https://discord.gg/vbE4MaJ"
  >
    Discord
  </a>
</footer>
`;
